### PR TITLE
Health-RI GDI contributions to fairdatapoint 

### PR DESCRIPTION
**Pull request by hcvdwerf**

### CODE REVIEW

**Comment by hcvdwerf** [comment]
Een pull request ingediend met de volgende aanpassingen:
- Toevoeging van unit tests en een bijbehorende pipeline.
- Oplossen van een UTF-8 bug.
- Code refactoring.
- Versie-update naar CKAN  2.10.
- Herschikking van code gerelateerd aan de Civity-harvester extensies.
Wij verzoeken u deze pull request te beoordelen en goed te keuren. Indien goedgekeurd een versie te releasen. Alvast bedankt!

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -7,12 +7,12 @@ jobs:
       # The CKAN version tag of the Solr and Postgres containers should match
       # the one of the container the tests run on.
       # You can switch this base image with a custom image tailored to your project
-      image: openknowledge/ckan-dev:2.9
+      image: openknowledge/ckan-dev:2.10
     services:
       solr:
         image: ckan/ckan-solr-dev:2.9
```
Comment (HilariousGil) [inline .github/workflows/test.yml]: Noticed the solr has not been updated to 2.10 as well, any reason why this image has not been updated as well?
Just checking if this was intentional or it just passed unoticed.
There are 2.10 versions available --> https://github.com/ckan/ckan-solr

File: README.md [hunk 2]
```diff
@@ -63,7 +63,7 @@ To install ckanext-fairdatapoint:
 
 2. Clone the source and install it on the virtualenv
 
-    git clone https://github.com/CivityNL/ckanext-fairdatapoint.git
+    git clone https://github.com/GenomicDataInfrastructure/gdi-userportal-ckanext-fairdatapoint.git
```
Comment (HilariousGil) [inline README.md]: This should be kept as CivityNL in our repository.

**Comment by HilariousGil** [comment]
We also just publish the previous state of the extension as the [v0.0.1 version ](https://github.com/CivityNL/ckanext-fairdatapoint/releases/tag/0.0.1) and then once we merge we can release a new version.

File: .github/workflows/test.yml (hunk 1 above)
Comment (HilariousGil) [reply .github/workflows/test.yml]: Also noticed that the GitHub Actions are failing due to the image `ckan/ckan-solr-dev:2.9`  no longer being available.
See here https://hub.docker.com/search?q=ckan%2Fckan-solr you will get a 404.

(This is also an issue in our repository)

**Comment by HilariousGil** [comment]
This will be merged and a tag will be created.

Outside of the scope of this PR the following actions will be done shortly:

- Adding a [CONTRIBUTING.md](http://contributing.md/)  file
- Adding a [CODE_OF_CONDUCT.md](https://github.com/ckan/ckan/blob/master/CODE_OF_CONDUCT.md) file 
- File Headers: Update with copyrights and licenses

**Pull request merged**